src/components/standup/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { FC, useState, useEffect } from 'react';
+import moment from 'moment';
+import styles from '@/components/standup/standupContainer.module.scss';
+
+import {
+    useAddStandupMutation,
+    useUserStandupDetailsQuery,
+} from '@/app/services/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+
+import FormInputComponent from './FormInputComponent';
+import { standupUpdateType } from '@/types/standup.type';
+import { getTotalMissedUpdate } from '@/utils/getTotalMissedUpdate';
+import { toast, ToastTypes } from '@/helperFunctions/toast';
+
+const StandUpContainer: FC = () => {
+    const [standupUpdate, setStandupUpdate] = useState<standupUpdateType>({
+        type: 'user',
+        completed: '',
+        planned: '',
+        blockers: '',
+    });
+
+    const [buttonDisable, setButtonDisable] = useState<boolean>(true);
+    const [addStandup] = useAddStandupMutation();
+    const { data: user } = useGetUserQuery();
+    const { data: userStandupdata } = useUserStandupDetailsQuery(user?.id);
+
+    const { SUCCESS, ERROR } = ToastTypes;
+    const standupDates = userStandupdata?.data?.map((element) => element.date);
+    const totalMissedUpdate = getTotalMissedUpdate(standupDates || []);
+    const yesterdayDate = moment().subtract(1, 'days').format('MMMM DD, YYYY');
+
+    const buttonStyleClass = buttonDisable
+        ? `${styles.nonActiveButton}`
+        : `${styles.activeButton}`;
+
+    const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+        setStandupUpdate((prevStandupUpdate) => ({
+            ...prevStandupUpdate,
+            [event.target.name]: event.target.value,
+        }));
+    };
+    const isValidate = () => {
+        return (
+            standupUpdate.completed !== '' &&
+            standupUpdate.planned !== '' &&
+            standupUpdate.blockers !== ''
+        );
+    };
+    useEffect(() => {
+        const isValid = isValidate();
+        setButtonDisable(!isValid);
+    }, [standupUpdate.completed, standupUpdate.planned]);
+
+    const handleFormSubmission = async (
+        event: React.FormEvent<HTMLFormElement>
+    ) => {
+        event.preventDefault();
+        try {
+            await addStandup(standupUpdate);
+            toast(SUCCESS, 'Standup submitted successfully');
+            setStandupUpdate({
+                type: 'user',
+                completed: '',
+                planned: '',
+                blockers: '',
+            });
+        } catch (error) {
+            toast(ERROR, 'Something went wrong!');
```

Comment:
Making constants for the error messages is a brilliant idea. It will not only help us maintain a standardized approach but also make troubleshooting and debugging easier.

Reply:
I think it's good to switch to better quality when prompted to. (add this to your feature and create an issue ticket that will take care of it on other files) we can assign that to someone then.
> Make constants for the error messages

thanks @sahsisunny for pointing it out.

Reply:
please add console.error and log the error